lua/nvim-tree/appearance.lua

```diff
@@ -191,8 +267,10 @@ function M.setup()
   end
 
   -- default links
-  for from, to in pairs(DEFAULT_LINKS) do
-    vim.api.nvim_command("hi def link " .. from .. " " .. to)
+  for _, g in ipairs(HIGHLIGHT_GROUPS) do
+    if g.link then
+      vim.api.nvim_command("hi def link " .. g.group .. " " .. g.link)
+    end
```

Comment:
Order was preserved here: defined, legacy, links

Erring on the side of caution.